lighthouse-core/lib/styles-helpers.js

```diff
@@ -96,14 +96,14 @@ function getFormattedStyleRule(content, parsedContent) {
     }, []).join('\n');
   }
 
-  const block = `
-${parsedContent.selector} {
+  const block = `\`\`\`${parsedContent.selector} {
```

Comment:
doing this in the `extendedInfo` value seems a bit ugly. What about a different keyword for the table formatter that annotates code blocks with the ```s?